before_install.sh

```diff
@@ -59,8 +59,8 @@ fi)
 
 #TODO: work out how to avoid the need for this
 chmod u+x packages/FSharp.Compiler.Tools.4.1.5/tools/fsi.exe 
```

Comment:
We should really consider to use a dependency manager that doesn't have version numbers in path